src/Codeception/Lib/Driver/Db.php

```diff
@@ -294,6 +294,8 @@ public function executeQuery($query, array $params): PDOStatement
                 $type = PDO::PARAM_BOOL;
             } elseif (is_int($param)) {
                 $type = PDO::PARAM_INT;
+            } elseif ($this->isBinary($param)) {
```

Comment:
This change leads to a type error if `$param === null`

I just created https://github.com/Codeception/module-db/pull/63 to fix this issue.